upgrade/breaking-changes/3-0-0.md

```diff
@@ -16,7 +16,7 @@ position: 1
 - TreeView - add `ExpandedItems` parameter and remove `ExpandedField`
 - Grid – change `OnRead` method to return the `Data`. This is an important change that will empower us to deliver more features in the future(like the export of all Grid pages), and the Grid will not rely on component lifecycle for getting the data.
 - Grid – use `IFilterDescriptor` in GridState instead of `FilterDescriptorBase`
-- DateInput – remove `ParsingErrorMessage` in favor of localization string for the error message
+- DateInput – remove `ParsingErrorMessage` (obsolete since 2.8) in favor of `DateInput_ParsingErrorMessage` [localization string]({%slug globalization-localization%}).
```

Comment:
```suggestion
- DateInput – remove `ParsingErrorMessage` (obsolete since version 2.8) in favor of `DateInput_ParsingErrorMessage` [localization string]({%slug globalization-localization%}).
```